config/develop/bridgeserver2.yaml

```diff
@@ -4,32 +4,44 @@ depedencies:
   - common/bridgeserver2-common.yaml
   - develop/bridgeserver2-db.yaml
 parameters:
-  AppDeployBucket: org-sagebridge-bridgeserver2-deployment-develop
+  AppDeployBucket: org-sagebridge-bridgeserver2-deployment-devdevelop
   AppHealthcheckUrl: '/'
-  AttachmentBucket: org-sagebridge-attachment-develop
+  AttachmentBucket: org-sagebridge-attachment-devdevelop
   AwsAutoScalingMaxSize: '1'
   AwsAutoScalingMinSize: '1'
   AwsEbHealthReportingSystem: enhanced
   AwsSnsBounceNotificationEndpoint: bridgepf-develop-bounce-notifications@sagebase.org
   AwsSnsNotificationEndpoint: bridgepf-develop@sagebase.org
+  BootstrapAdminEmail: !ssm /bridgeserver2-develop/BootstrapAdminEmail
+  BootstrapAdminPassword: !ssm /bridgeserver2-develop/BootstrapAdminPassword
```

Comment:
what are the origin of these credentials, synapse or aws?  Can we use a IAM role instead?

Reply:
The credentials come from Bridge and are used by the Bootstrapper to create Bridge admin accounts. There's not really any other place to put these.